src/pages/Interferometry/MachZehnderCanvas.js

```diff
@@ -2,21 +2,98 @@ import React, { useRef } from "react";
 import PropTypes from "prop-types";
 import useAnimationFrame from "../../useAnimation";
 import MachZehnderEquipment from "./MachZehnderEquipment";
+import Photon from "./Photon";
 
 const equipment = new MachZehnderEquipment(
   100,
   100,
-  400,
+  800,
   400,
 );
 
-function MachZehnderCanvas({ size: { width, height } }) {
+const generateNumberOfPhotons = (number) => {
+  const photonArray = [];
```

Comment:
`photons` is better because it's shorter and cleaner.
`photonArray` seems too redundant.